frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageInfoCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <div class="d-flex justify-end flex-wrap align-stretch">
+      <RecipePageInfoCardImage v-if="landscape" :recipe="recipe" />
+      <v-card
+        :width="landscape ? '100%' : '50%'"
+        flat
+        class="d-flex flex-column justify-center align-center"
+      >
+        <v-card-text>
+          <v-card-title class="headline pa-0 flex-column align-center">
+            {{ recipe.name }}
+            <RecipeRating :key="recipe.slug" :value="recipe.rating" :recipe-id="recipe.id" :slug="recipe.slug" />
+          </v-card-title>
+          <v-divider class="my-2" />
+          <SafeMarkdown :source="recipe.description" />
+          <v-divider />
+          <v-container class="d-flex flex-row flex-wrap justify-center align-center">
+            <div>
+              <v-row no-gutters class="mb-1">
+                <v-col v-if="recipe.recipeYieldQuantity || recipe.recipeYield" cols="12" class="d-flex flex-wrap justify-center">
+                  <RecipeYield
+                    :yield-quantity="recipe.recipeYieldQuantity"
+                    :yield="recipe.recipeYield"
+                  />
```

Comment:
This does currently not scale when increasing/decreasing the servings. 
It seems like the component already supports it, so we should probably make use of it.

Reply:
I can add that back in, but it was a conscious choice on my part to omit it since it's no longer near the scale controls. To me it reads "this recipe normally yields X Amount" and then the scale over by the ingredients says "this is what it's set to right now"

But maybe I'm overthinking it and it should just scale everywhere